src/Auth0.AuthenticationApi/Tokens/SymmetricSignatureVerifier.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Microsoft.IdentityModel.Tokens;
+using System;
+using System.IdentityModel.Tokens.Jwt;
+using System.Text;
+
+namespace Auth0.AuthenticationApi.Tokens
+{
+    internal class SymmetricSignatureVerifier : ISignatureVerifier
+    {
+        private readonly SecurityKey signingKey;
+
+        public SymmetricSignatureVerifier(SecurityKey signingKey)
+        {
+            this.signingKey = signingKey;
+        }
+
+        public static SymmetricSignatureVerifier FromClientSecret(string clientSecret, bool isBase64Encoded = false)
```

Comment:
`isBase64Encoded ` is legacy (+3 years old setting). Please remove if this hits a new major.